LICENSE

```diff
@@ -186,7 +186,7 @@ APPENDIX: How to apply the Apache License to your work.
    same "printed page" as the copyright notice for easier
    identification within third-party archives.
 
-Copyright 2019 Exonum Team
+Copyright 2020 Exonum Team
```

Comment:
Off-topic: I don't think we're supposed to change the license contents; this part just provides an example how to apply the license (as we do in the license header). See, e.g., [the license text](https://github.com/rust-lang/rust/blob/766fba3fdca8fe075c370a63d2f76f8407483af5/LICENSE-APACHE#L189) in the Rust lang repo.